moztelemetry/partition-packing/README.md

```diff
@@ -0,0 +1,36 @@
+# Partition Packing Performance Measures
+
+This group of tests measures loading strategies for moztelemetry datasets. Partitions are made according to two different strategies, default behavior and equally sized groups.
+
+## Background:
+We set up the tests in Scala and Python and compare the results of default
+behavior and equally sized groups in each language. The tests measure how long
+it takes to partition the data. Dataset, channel, the submission date are all
```

Comment:
While being pendantic, it seems to me that the number and size of the files are the variables we really care about.  The doc-type, channel, and submission date are knobs that can be used to adjust those variables. For example, nightly has less rows but also contains the [pre-release histograms](https://telemetry.mozilla.org/probe-dictionary/?optout=true&channel=nightly&view=stats-view). Release has many more rows, but less data per row than nightly.